test/controllers/clinics_controller_test.rb

```diff
@@ -21,6 +21,16 @@ class ClinicsControllerTest < ActionDispatch::IntegrationTest
     end
```

Comment:
An additional test in this controller to confirm that it responds to `index` as json would be great! Just a ping to confirm that the endpoint returns JSON data roughly in the format you're consuming.